src/returns/lines.js

```diff
@@ -3,6 +3,12 @@ const naldDates = require('../nald/dates');
 
 const formatMoment = date => date.format('YYYY-MM-DD');
 
+/**
```

Comment:
If this only applies to months maybe we could name it something more descriptive, e.g. `isLastDayOfMonth`.
There is a docblock but the arguments and return are not documented - perhaps we should either add comments or remove it.
Could we also check whether monthly test cases include all possible combinations of:
- Is last day of month: yes/no
- Is final return: yes/no